### PR TITLE
Fix codespell

### PR DESCRIPTION
**Pull request by polybassa**
Apparently, codespell received an update.

### CODE REVIEW

File: test/scapy/layers/tls/tls.uts [hunk 1]
```diff
@@ -1521,7 +1521,7 @@ assert [type(x) for x in a.msg] == [TLSServerHello, TLSCertificate, TLSServerKey
...
...9\xb3!\x02\x03\x01\x00\x01\xa3S0Q0\x1d\x06\x03U\x1d\x0e\x04\x16\x04\x14\x82\x97\xfb\x8c\xf3\xd9\xdd\xf2\xf6\x0e\xc9\x0f\x92\x81\\(\xe3\xf3\xff50\x1f\x06\x03U\x1d#\x04\x180\x16\x80\x14\x82\x97\xfb\x8c\xf3\xd9\xdd\xf2\xf6\x0e\xc9\x0f\x92\x81\\(\xe3\xf3\xff50\x0f\x06\x03U\x1d\x13\x01\x01\xff\x04\x050\x03\x01\x01\xff0\r\x06\t*\x86H\x86\xf7\r\x01\x01\x0b\x05\x00\x03\x82\x01\x01\x00<\x00\x1f\x0f]\x10`r\xe0\xbe\xed\xfaG\x89S)\xd4b4"\x08\x0ef\xca\xa4\xbe\xb4\xc3\xd9\xb6\x86\x8f\xa1\x07F}a`\xe5\x12\xed\xe7\xcb\xbe\xfd\xeb\x17\xc0\x9b\x9fYO\x86\xf1\xa9\x92,\x98,\xcf\x96U\xd4\xd6~\xda\x06\x1ff~\xeb%\xfer\x03\xbe\xa0\x0e@\xf1P\xa4\x90\xa7\x8c\xa9c\xd8|\x18Z\xde\x8b|)OPR\xfb\x1e\x1e\xdb4\x03\x83\x1e3\xe4\x02l\x8eVq|\xb7{\xc3#!\x85\x8b\xe3zw\xfe\x7fU\x11\xef\x8c \x13\xc8n\'0\xc5\xb26hu\x13\x1c\xae\x0cv\x16\xfd\xc7\xc8`V\x96\x13>zh_  <\r\xb8\xe0\xff\x1d\x1aY\x91\xd2\xf0X\xf4\x8f \xb1\n_\xb0\xdf\'\xa1\xf9\x87L\xc0\xfe\x8dn\xbfw\xe9\xa7\xba8I\x0e\x9dc$\x1a\x0f\xb3\xfdw\x01\xff;\x13\x0c\x9a\xa7\xaaww\x02\x80\xb7\x00<\x1b\xb5\xe0xL4\xaa\xcbt\xce\x81\x14\x96\x0eP\xee\xe0F\x02\xa7\xab \xe5\xc8x\x02\x8eB\x92\xe9\x0e@\xfdc\x1f\xee\x16\x03\x03\x00\x04\x0e\x00\x00\x00', tls_session=r1.tls_session.mirror())
```
Comment (guedou) [inline test/scapy/layers/tls/tls.uts:1524]: What is the difference here?

**Review by guedou — COMMENTED** [review]
Thanks for the PR. It is fun to see that typos are fixed over time.

File: test/scapy/layers/tls/tls.uts (hunk 1 above)
Comment (polybassa) [reply test/scapy/layers/tls/tls.uts:1524]: Internet Widgits Pty Ltd => Internet Widgets Pty Ltd

Comment (polybassa) [reply test/scapy/layers/tls/tls.uts:1524]: dunno if this change makes sense... ;-D

Comment (guedou) [reply test/scapy/layers/tls/tls.uts:1524]: This is not a typo =) I think that we should not change it.

Comment (polybassa) [reply test/scapy/layers/tls/tls.uts:1524]: but does it hurt to change it? Otherwise we need an exception for codespell

Comment (guedou) [reply test/scapy/layers/tls/tls.uts:1524]: Well, you are likely changing the content of a X.509 certificate and thus breaking the associated signature. That change might back-fire someday.

Comment (polybassa) [reply test/scapy/layers/tls/tls.uts:1524]: ok ;-D sounds legit

**Review by guedou — APPROVED** [review]

File: test/contrib/automotive/ccp.uts [hunk 2]
```diff
@@ -903,7 +903,7 @@ assert hasattr(dto, "load") == False
 assert dto.MTA0_extension == 2
 assert dto.MTA0_address == 0x34002006
 
-= CAN Socket sr1 with dto.ansers(cro) == False
+= CAN Socket sr1 with dto.answers(cro) == False
```
Comment (gpotter2) [inline test/contrib/automotive/ccp.uts:906]: How did this work before?

**Comment by polybassa** [comment]
It’s just the name of the testcase

--
Mit freundlichen Grüßen

Dr. Nils Weiß

dissecto GmbH
Franz-Mayer-Str. 1
93053 Regensburg
________________________________

File: test/contrib/automotive/ccp.uts (hunk 2 above)
Comment (gpotter2) [reply test/contrib/automotive/ccp.uts:906]: IMG_1824

Sorry I'm dumb, but the layout was unlucky.

**Pull request merged**